test/matchers/mock/toBeRequestedTimes.test.ts

```diff
@@ -83,21 +83,21 @@ describe('toBeRequestedTimes', () => {
 
         // expect(mock).not.toBeRequestedTimes(0) should fail
         const result = await toBeRequestedTimes.call({ isNot: true }, mock, 0)
-        expect(result.pass).toBe(true)
+        expect(result.pass).toBe(false)
 
         // expect(mock).not.toBeRequestedTimes(1) should pass
         const result2 = await toBeRequestedTimes.call({ isNot: true }, mock, 1)
-        expect(result2.pass).toBe(false)
+        expect(result2.pass).toBe(true)
 
         mock.calls.push(mockMatch)
 
         // expect(mock).not.toBeRequestedTimes(0) should pass
         const result3 = await toBeRequestedTimes.call({ isNot: true }, mock, 0)
-        expect(result3.pass).toBe(false)
+        expect(result3.pass).toBe(true)
 
         // expect(mock).not.toBeRequestedTimes(1) should fail
         const result4 = await toBeRequestedTimes.call({ isNot: true }, mock, 1)
-        expect(result4.pass).toBe(true)
+        expect(result4.pass).toBe(false)
```

Comment:
Why are all these test results reversed? The matcher doesn't seem to be related with your change.

Reply:
Following on from my comment below about why I changed waitUntil, I then looked at some of the old tests for isNot and they seemed to be the wrong way around and the comment here seems to agree.

Taking line 94 for example:

`expect(mock).not.toBeRequestedTimes(0) should pass`

Because we have online 92 pushed mockMatch then we expect in the positive case toBeRequestedTimes(1) would pass.

Furthermore .toBeRequestedTimes(0) is false and finally .not.toBeRequestedTimes(0) is true? Again happy to be wrong on this but it seems to me that it was the wrong way around prior to the update to waitUntil